{
    "config": {
        "abort": {
            "already_configured": "Slu\u017eba je ji\u017e nastavena",
            "reauth_successful": "Op\u011btovn\u00e9 ov\u011b\u0159en\u00ed bylo \u00fasp\u011b\u0161n\u00e9"
        },
        "error": {
            "cannot_connect": "Nepoda\u0159ilo se p\u0159ipojit",
            "invalid_api_key": "Neplatn\u00fd kl\u00ed\u010d API",
            "invalid_user_key": "Neplatn\u00fd u\u017eivatelsk\u00fd kl\u00ed\u010d"
        },
        "step": {
            "reauth_confirm": {
                "data": {
                    "api_key": "Kl\u00ed\u010d API"
                },
                "title": "Znovu ov\u011b\u0159it integraci"
            },
            "user": {
                "data": {
                    "api_key": "Kl\u00ed\u010d API",
                    "name": "N\u00e1zev",
                    "user_key": "U\u017eivatelsk\u00fd kl\u00ed\u010d"
                }
            }
        }
    },
    "issues": {
        "removed_yaml": {
            "description": "Konfigurace Pushover pomoc\u00ed YAML byla odebr\u00e1na.\n\nVa\u0161e st\u00e1vaj\u00edc\u00ed konfigurace YAML nepou\u017e\u00edv\u00e1 Home Assistant. \n\nOdeberte konfiguraci Pushover YAML ze souboru configuration.yaml a restartujte Home Assistanta, abyste tento probl\u00e9m vy\u0159e\u0161ili.",
            "title": "Konfigurace Pushover YAML byla odebr\u00e1na"
        }
    }
}